examples/air-quality-sensor-app/air-quality-sensor-common/include/air-quality-sensor-manager.h

```diff
@@ -1,11 +1,9 @@
-#include <app-common/zap-generated/ids/Attributes.h>
-#include <app-common/zap-generated/ids/Clusters.h>
```

Comment:
Why are the id includes being removed?  This file uses those ids directly, so it should include the headers they are defined in; basic include-what-you-use.